tls/s2n_config.c

```diff
@@ -1071,3 +1072,19 @@ int s2n_config_set_recv_multi_record(struct s2n_config *config, bool enabled)
 
     return S2N_SUCCESS;
 }
+
+/* Indicates if the connection should attempt to enable kTLS. */
+int s2n_config_ktls_enable(struct s2n_config *config, s2n_ktls_mode ktls_mode)
+{
+    POSIX_ENSURE_REF(config);
+
+    config->ktls_mode_requested = ktls_mode;
```

Comment:
Can ktls_mode be S2N_KTLS_MODE_DISABLED? Would a better name for this be s2n_config_set_ktls_mode?